<header class="banner"><div class="banner">
  <div class="logos">
    <div><img title="Julia logo" src="../figures/julia-logo-color.svg" alt="Julia animated logo" width=35 /></div>
    <div>◌</div>
    <div><img title="Powered by MathJax" src="https://www.mathjax.org/badge/mj-logo.svg"
              alt="Powered by MathJax" width=65 />
    </div>
  </div>  <!-- logos -->
  <div class="cite_textbook cb8">3.028 SI <span class="authors"></span></div>
</div></header>




<footer class="navbar navbar-default navbar-fixed-bottom mcs-psy-nav" >
  <div class="container-fluid text-right h4">
    <em>ψᵟ</em>
  </div>
</footer> 

### Given 
A 1.8 m<sup>3</sup> rigid tank contains steam at 220 °C. One-third of the volume is in the liquid phase and the rest is in the vapor form. 

### Find
1. the pressure of the steam, 
1. the quality of the saturated mixture, and 
1. the density of the mixture.

### Get Organized

1. We are told we have both liquid and vapor phases present. **Two phases can only coexist under saturation condictions.** Immediately, we know that our pressure will be the saturation pressure at our given temperature.   
2. We need quality. Quality $x$ is a mass ratio,  
 \begin{gather} 
  \notag \\
  x = \frac{m_\ce{vapor}}{m_\ce{total}}, \\ {} \notag
 \end{gather}
 but we are given a volume ratio. However, we know that specific volume relates volume and mass via  
 \begin{gather}
  \notag \\
  v = \frac{V}{m}.
 \end{gather}
 Thus, 
 \begin{gather} 
  \notag \\
  x = \frac{m_\ce{vapor}}{m_\ce{total}} = \frac{m_\ce{g}}{m_\ce{f} + m_\ce{g}} ,
 \end{gather} 
 where
 \begin{gather} 
  \notag \\
  m_\ce{g} = \frac{V_\ce{g}}{v_\ce{g}} = \frac{\frac{2}{3}V}{v_\ce{g}}, 
 \end{gather} 
 and
 \begin{gather} 
  \notag \\
  m_\ce{f} = \frac{V_\ce{f}}{v_\ce{f}} = \frac{\frac{1}{3}V}{v_\ce{f}}. 
 \\ \notag {}
 \end{gather}  
2. We have multiple paths for calculating density. We can use it's definition 
 \begin{gather} 
  \notag \\
  \rho = \frac{m}{V} = \frac{m_\ce{f} + m_\ce{g}}{V}. 
 \end{gather}  
$$\sf {OR}$$   
We can calculate density from specific volume, with    
 \begin{gather} 
  \notag \\
  \rho = \frac{1}{v}, 
 \end{gather}  
 and 
 \begin{gather} 
  \notag \\
  v = v_\ce{f} + x \left(v_\ce{g} - v_\ce{f} \right)
 \end{gather}  

### Attack

Let's get to work.

#### 1. Steam pressure.  
Get it right off the saturated steam temperature table.  
<div class="panel-success"><div class="panel-heading"><div class="badge"></div> 
\begin{gather*} P = P_\rm{sat@220℃} = 23.18\, \ce{bar}
\end{gather*}
</div></div>

#### 2. Quality.
We developed an expression for quality in eqn's (3), (4), and (5). We need $v_\ce{f}$ and $v_\ce{g},$ both evaluated at 220 °C. Again, we go to the saturated steam temperature table.


In [1]:
# ⋮⋮⋮⋮⋮ Julia initializations and setups ⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮⋮
using Unitful     # units package for Julia

V = 1.8u"m^3"     # Unitful quatity, units declaration

1.8 m^3

In [2]:
vf_220°C = 0.00119u"m^3/kg"  # Julia allows unicode symbols in variable names.
vg_220°C = 0.08619u"m^3/kg"  # Python does also, but its support is limited.

mg = 2/3 * V / vg_220°C      # eqn (4)
mf = 1/3 * V / vf_220°C      # eqn (5)

x = mg / (mf + mg)           # eqn (3)
@show x;

0.02687140115163148

<div class="panel-success"><div class="panel-heading"><div class="badge"></div> 
\begin{gather*} 
    x = 0.0269
\end{gather*}
</div></div>

#### 3. Density.


In [3]:
# Density via eqn (6)

ρ = (mf + mg) / V 

println("ρ_eqn(6) = ",ρ)

ρ_eqn(6) = 287.84689418173576 kg m^-3


In [4]:
# Density via eqns (7), (8)

v = vf_220°C + x*(vg_220°C - vf_220°C)

println("ρ_eqn(7) = ",1/v)

ρ_eqn(7) = 287.84689418173576 kg m^-3


<div class="panel-success"><div class="panel-heading"><div class="badge"></div> 
\begin{gather*} 
    \rho_\rm{eqn(6)} = \rho_\rm{eqn(7)} = 287.85\, kg/m^3
\end{gather*}
</div></div>

<footer style="background-color: #461d7c; color:rgba(253,208,35,0.35); line-height:2em; padding-left:1em; padding-right:0.5em; bottom:0; float:bottom; text-align:right;"><i>ψᵟ</i></footer>